From 79beb584c2d81fccb5cb8759ad0813ac0544c4dc Mon Sep 17 00:00:00 2001
From: tkimball83 <tkimball83@users.noreply.github.com>
Date: Wed, 28 Feb 2018 03:37:26 -0800
Subject: [PATCH 2/9] Fix: Start medusa service after network.target (#3816)

---
 runscripts/init.systemd | 1 +
 1 file changed, 1 insertion(+)

diff --git a/runscripts/init.systemd b/runscripts/init.systemd
index c3d0b5a..841fd87 100755
--- a/runscripts/init.systemd
+++ b/runscripts/init.systemd
@@ -44,6 +44,7 @@
 
 [Unit]
 Description=Medusa Daemon
+After=network.target
 
 [Service]
 User=medusa
-- 
2.6.3.windows.1

